pkg/segment/aggregations/streamstats.go

```diff
@@ -247,9 +303,9 @@ func cleanTimeWindow(currTimestamp uint64, timeSortAsc bool, timeWindow *structs
 	return nil
 }
 
-func getResults(ssResults *structs.RunningStreamStatsResults, measureAgg utils.AggregateFunctions) (float64, bool, error) {
+func getResults(ssResults *structs.RunningStreamStatsResults, measureAgg utils.AggregateFunctions) (interface{}, bool, error) {
```

Comment:
I'd be cautious of returning an `interface{}`because it's more general than you need (so it takes people a little longer to figure out exactly what types can actually be returned from this method), and it complicates how the caller will extract the value.

Is there a good way to return something more specific, that still allows you to return multiple types of values? If there isn't, you can keep this

Reply:
Yeah, I didn't find anything perfectly suitable. Moreover, there may be other types of aggregations that are yet to be implemented which could return different kinds of values. I think making it generic would be better since the record too contains interface{} as the column value and the output of streamstats is also a column value.